src/cli/cli.cpp

```diff
@@ -3687,6 +3688,63 @@ void Interpreter::ProcessUdp(uint8_t aArgsLength, char *aArgs[])
     AppendResult(error);
 }
 
+void Interpreter::ProcessUnsecurePort(uint8_t aArgsLength, char *aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(aArgsLength >= 1, error = OT_ERROR_INVALID_ARGS);
+
+    if ((aArgsLength == 2) && (strcmp(aArgs[0], "add") == 0))
```

Comment:
If the user types `unsecureport add`, it will be given error `OT_ERROR_INVALID_COMMAND`. But the error should be `OT_ERROR_INVALID_ARGS`.

Reply:
Updated